packages/core/src/decorators/attachment/Attachment.ts

```diff
@@ -45,6 +47,16 @@ export class AttachmentData {
     }
   }
 
+  public getDataAsJson() {
```

Comment:
Would it be a good idea to add a generic here? This would allows us to get the typing from the return type in a nice way.

Reply:
probably 😅 will have to have a quick read into how that works in TS as I'm a noob.

Reply:
But not sure as there is no args (input)

Reply:
not tested but this should work

```ts
public getDataAsJson<T>(): T {
return json as T
}
```